package.json

```diff
@@ -99,16 +99,14 @@
         "jest": "^26.4.2",
         "jest-config": "^26.4.2",
         "jest-environment-jsdom": "^26.6.2",
-        "lint-staged": "^11.1.1",
-        "prettier": "^2.3.2",
```

Comment:
why you are removing them?

Reply:
seems like package.json is automerged and couple dependencies gets lost, added them in https://github.com/energywebfoundation/iam-client-lib/pull/226/commits/8ba9bff2d5294634acf6ce771029da6e0b2f7fe5

please check